install/admiral/overlays/demosinglecluster/envconfig_values.yaml

```diff
@@ -19,4 +19,4 @@ spec:
             - admiral-sync
             - --sync_period
             - 20s
-          name: admiral
\ No newline at end of file
+          name: admiraladmiral/pkg/controller/admiral/service_test.go
```

Comment:
An accidental change?

Reply:
Thanks for the careful review. Yes, that was accidental